### PR TITLE
Added 02.echo-bot

### PR DESCRIPTION
**Pull request by tracyboehrer**

### CODE REVIEW

File: samples/02.echo-bot/config.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+""" Bot Configuration """
+
+
+class DefaultConfig(object):
+    """ Bot Configuration """
+
+    PORT = 3978
+    APP_ID = os.environ.get("MicrosoftAppId", "")
+    APP_PASSWORD = os.environ.get("MicrosoftAppPassword", "")
+    LUIS_APP_ID = os.environ.get("LuisAppId", "")
```
Comment (daveta) [inline samples/02.echo-bot/config.py]: >LUIS_APP_ID = os.environ.get("LuisAppId", "") [](start = 4, length = 45)

Are these needed?

Comment (tracyboehrer) [reply samples/02.echo-bot/config.py]: Nope.  Removed.

File: samples/02.echo-bot/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+git clone https://github.com/Microsoft/botbuilder-python.git
+```
+- Run `pip install -r requirements.txt` to install all dependencies
+- Run `python app.py`
+- Alternatively to the last command, you can set the file in an environment variable with `set FLASK_APP=app.py` in windows (`export FLASK_APP=app.py` in mac/linux) and then run `flask run --host=127.0.0.1 --port=3978`
+
+
+### Visual studio code
+- Activate your desired virtual environment
+- Open `botbuilder-python\samples\45.state-management` folder
```
Comment (axelsrz) [inline samples/02.echo-bot/README.md]: Please remove this line

File: samples/02.echo-bot/adapter_with_error_handler.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+class AdapterWithErrorHandler(BotFrameworkAdapter):
+    def __init__(
+        self,
+        settings: BotFrameworkAdapterSettings
+    ):
+        super().__init__(settings)
+
+        # Catch-all for errors.
+        async def on_error(context: TurnContext, error: Exception):
```
Comment (axelsrz) [inline samples/02.echo-bot/adapter_with_error_handler.py]: Please move this to app.py and attach it directly to the instance

**Pull request merged**